deployments/grafana-demo/README.md

```diff
@@ -0,0 +1,23 @@
+# How to deploy
```

Comment:
Can we split this demo example to its own PR?

Reply:
Done! Here is the PR: https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/37